README.md

```diff
@@ -5,57 +5,166 @@
+[Picard](http://broadinstitute.github.io/picard/) codebases under a single simplified, streamlined framework,
+and to enable selected tools to be run in a massively parallel way on local clusters or in the cloud using
+[Apache Spark](http://spark.apache.org/).
+
+This project is in an alpha development stage and is not yet ready for general use.
 
 If you are looking for the current version of GATK to use in production work, please see the [GATK website](http://www.broadinstitute.org/gatk), where you can download a precompiled executable, read documentation, ask questions and receive technical support.
 
 If you are looking for the codebase of the current production version of GATK, please see either the [GATK development framework repository](https://github.com/broadgsa/gatk/) or the [full GATK tools repository](https://github.com/broadgsa/gatk-protected/).
 
-
 ##Requirements
 * Java 8
-
-* Gradle 2.7
-
-* R 3.1.3
-
-* Git 2.5
-
-* [git-lfs](https://git-lfs.github.com/) 1.0.2 is needed to to download large files for the complete test suite.  
-Run `git lfs init` after installing, followed by `git lfs pull` to download the large files.   (The download is ~500 MB)
-
-
-##Installation
-To build and run all tests, run `gradle check`. Test report is in `build/reports/tests/index.html`.
-
-To only build, run `gradle installDist`.
-
-To run all tests, run `gradle test`. 
-What will happen depends on the value of the `CLOUD` environment variable: if it's `false` or unset then only local tests are run. If it's `mandatory` then it'll run only the cloud tests. 
-
-To run a single test class, run something like this, `gradle test -Dtest.single=ReadUtilsUnitTest`.
-
-To run tests and compute coverage reports, run `gradle jacocoTestReport`. The report is then in `build/reports/jacoco/test/html/index.html`. (IntelliJ 14 has a good coverage tool that is preferable for development).
-
-
-Note: for faster gradle operations, add `org.gradle.daemon=true` to your `~/.gradle/gradle.properties` file.  This will keep a gradle daemon running in the background and avoid the ~6s gradle start up time on every command.  
+* Git 2.5 or greater
+* Optional, but recommended:
+    * Python 2.6 or greater (needed for running the `gatk-launch` frontend script)
+    * Gradle 2.7 or greater (recommended, but can use the `./gradlew` script instead which will download an appropriate gradle version for you)
+    * R 3.1.3 (needed for producing plots in certain tools, and for running the test suite)
+    * [git-lfs](https://git-lfs.github.com/) 1.1.0 or greater (needed to download large files for the complete test suite).
+      Run `git lfs install` after downloading, followed by `git lfs pull` to download the large files. The download is ~500 MB.
+
+##Quick Start Guide
+
+* Build the GATK: `gradle installAll`
+* Get a list of available tools: `./gatk-launch --help`
+* Run a tool: `./gatk-launch PrintReads -I input.bam -O output.bam`
```

Comment:
I'd prefer not to do this, as it would greatly increase the length of the commands, and decrease readability. We're stuck with pseudo-inputs anyway for many of the commands due to things like `--cluster`